src/utils/positionUtils.ts

```diff
@@ -314,10 +328,15 @@ export const calcTradeFee = ({
   const takerAdiabaticFeeRate = Big6Math.mul(takerFee.adiabaticFee, Big6Math.div(skewDelta / 2n, takerFee.scale))
   const takerAdiabaticFee = Big6Math.mul(notional, takerAdiabaticFeeRate)
   const takerLinearFee = Big6Math.mul(notional, takerFee.linearFee)
-  const total = takerLinearFee + takerProportionalFee + takerAdiabaticFee
-  const feeBasisPoints = !Big6Math.isZero(total) ? Big6Math.div(total, notional) : 0n
+  const subtractiveFee = Big6Math.mul(takerLinearFee, referralFee)
+  const marketFee = Big6Math.mul(takerLinearFee - subtractiveFee, positionFee)
+  const tradeFee = subtractiveFee + marketFee
+  const feeBasisPoints = !Big6Math.isZero(tradeFee) ? Big6Math.div(tradeFee, notional) : 0n
```

Comment:
We'll need to make sure to update this calculation in the interface repo to not do the multiplication by the `parameter.positionFee`

https://github.com/equilibria-xyz/perennial-interface-v2/blob/8dca24afa21ba9508b83508b6fb23d97d2b6325e/src/components/pages/Trade/TradeForm/components/Receipt/index.tsx#L294